cloudinary-core/src/main/java/com/cloudinary/ArchiveParams.java

```diff
@@ -176,6 +177,15 @@ public ArchiveParams publicIds(String[] publicIds) {
         return this;
     }
 
+    public String[] fully_qualified_public_ids() {
```

Comment:
The method names in java must be camelCase (`fullyQualifiedPublicIds()`) even if the string in the hash is actually underscores.